all-together.ipynb

```diff
@@ -0,0 +1,775 @@
+{
```

Comment:
<p>Instead of adding the strings, you can use formatting: <code>f'grade-meteor-{name}'</code>  </p><p> </p><p>Seems like metrics computation is almost identical across different metrics. Let's make a single function that takes a metric (name and compute function) and applies it to the dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JetBrains-Research/metrics-evaluation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>